samples/tools/testbed/run_scenarios.py

```diff
@@ -0,0 +1,298 @@
+import os
+import errno
+import shutil
+import subprocess
+import json
+import sys
+import time
+import pathlib
+import argparse
+from autogen import config_list_from_json
+
+# Location of the environment directory
+INCLUDES_DIR = "./includes"
```

Comment:
Will this cause issues on Windows?

Reply:
Good question. I'll push a change without the "./"

Reply:
Done. Marking resolved.